vipps-partner-api.md

```diff
@@ -313,11 +316,14 @@ The user will then automatically be presented with the pre-filled PO.
    [portal.vipps.no](https://portal.vipps.no).
 2. The merchant uses the link and logs in with BankID on
    [portal.vipps.no](https://portal.vipps.no).
-3. The merchant is presented with a page informing them that they need to
+3. The merchant is presented with a text informing them that they need to
    sign an MA before filling in the PO.
-4. The merchant re-uses the link or finds the link to the pre-filled form on the
-   front page on
-   [portal.vipps.no](https://portal.vipps.no)
+   The merchant is provided with a link to the MA form, and the merchant must complete that
+   to be able to continue with the PO.
+4. After completing the MA form and submitting it,
+   the merchant re-uses the link to the pre-filled PO
+   (or finds the link to the pre-filled form on the front page on
```

Comment:
Maybe clarify that this is only possible if the ma is automatically approved, like you made me realize :sweat_smile:

Reply:
Hmm. Ok, that requires a rewrite, I think. Merging this now.